src/bloqade/squin/op/traits.py

```diff
@@ -5,22 +5,22 @@
 
 
 @dataclass(frozen=True)
-class Sized(ir.StmtTrait):
+class Sites(ir.StmtTrait):
     data: int
 
 
 @dataclass(frozen=True)
-class HasSize(ir.StmtTrait):
-    """An operator with a `size` attribute."""
+class HasNSitesTrait(ir.StmtTrait):
```

Comment:
Just `HasNSites`? I think `Trait` is redundant - just want to save some hand service time as old people :-)

Reply:
I agree it's pretty redundant but the problem I ran into last night was the change you proposed for the type lattice (from `OpShape` to `HasNSites`) collides with this unless I'm careful about fully qualifying both of them in code. If that's okay I can go ahead and just do that but otherwise this was the (admittedly rough) solution I came up with 😅

Reply:
Ah! I would use a longer name for the lattice then! Cuz that's unlikely used much